src/collections/_documentation/workflow/notifications/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Notifications
+sidebar_order: 2
+---
+
+Notifications in Sentry can be routed to many supported [integrations]({%- link _documentation/workflow/integrations/index.md-%}), but by default are aimed at email. Notifications fall into two categories:
+
+[Alerts]({%- link _documentation/workflow/notifications/alerts.md -%})
+
```

Comment:
I like how you've mad each of these headings (Alerts, Workflow) a link to the relevant page (it's nice and clean looking), but I worry a little that it's _too_ clean - that it won't be 100% obvious to folks that these are clickable. (I know, they're purple, but it's kinda subtle.) I'd be tempted to instead have a "Learn more" link after each blurb. What do you think?